StringOperations.cls

```diff
@@ -0,0 +1,30 @@
+public  class StringOperations {
+	
+	public static void stringOperations(String testString)
+	{
+		String replaceString = testString.replace('@','A'); 
+		system.debug('Replaced String:'+replaceString);	
+		
+		String substr = replaceString.substring(11); 
```

Comment:
@Eternus-noopur Why it's hardcoded index foe `substring` method. It should be dynamic.
- [ ] Remove hardcode index and make it dynamic.